packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -795,11 +795,11 @@ Future<bool> disableAnalytics() async {
   return await setAnalyticsEnabled(false);
 }
 
-Future<String?> getConsentMessage() async {
-  return await server.getConsentMessage();
+Future<String> fetchAnalyticsConsentMessage() async {
+  return await server.fetchAnalyticsConsentMessage();
 }
 
-Future<bool> confirmConsentMessageShown() async {
+Future<void> confirmConsentMessageShown() async {
```

Comment:
nit: "confirm" makes me think we are asking a question, or in other words that this would return a value of whether the message has been shown. Maybe `markConsentMessageAsShown` would make what this method does more clear?